Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.14
```

Comment:
let compile in `golang:1.14` and run in another image with smaller footprint, i.e. alpine, if possible at all